avm/res/network/virtual-network-gateway/tests/e2e/vpn/main.test.bicep

```diff
@@ -0,0 +1,155 @@
+targetScope = 'subscription'
+
+metadata name = 'VPN'
+metadata description = 'This instance deploys the module with the VPN set of required parameters.'
```

Comment:
Please keep in mind @fabmas for this and the other test files - if you say the below are the 'required' parameters for in this case VPN, that you don't have an army of optional parameters that are actually not required ;) It's quite confusing.

Reply:
Please re-review the test cases to make sure really only the parameters needed for the corresponding test case are included.

Reply:
@fabmas please don't resolve comments on your own unless they're suggestions. Otherwise it becomes very difficult to track which comments require re-reviewing ;)

That being said, my argument in the above comment stands so I'll add a couple more suggestions.